docs/source/develop.rst

```diff
@@ -22,6 +22,26 @@ and install it from source::
    cd distributed
    python setup.py install
 
+Using conda, for example::
+
+   git clone git@github.com:{your-fork}/distributed.git
+   cd distributed
+   conda create -y -n distributed python=3.6
+   conda activate distributed
+   python -m pip install -U -r requirements.txt
+   python -m pip install -U -r dev-requirements.txt
+   python -m pip install -e .
+
+To keep a fork in sync with the upstream source::
+
+   cd distributed
+   git checkout master
+   git remote add upstream git@github.com:dask/distributed.git
+   git remote -v
+   git fetch -ap upstream
```

Comment:
One last comment here, the `-p` may unintentionally remove tags the user would like to keep.  I don't think this is the common case, but if we are going to be recommending git commands perhaps we should avoid prunng?

Reply:
OK, better safe than sorry.  Done.